jenkins/ci.suse.de/templates/cloud-mkcloud-job-ha-ironic-template.yaml

```diff
@@ -30,5 +30,6 @@
             label={label}
             scenario=cloud{version}-ironic-basic-ha.yml
             job_name=cloud-mkcloud{version}-job-ha-ironic-{arch}
+            nics=2
             want_ironic=1
```

Comment:
Why solving this in a specific job, instead of in openstack-mkcloud?

My recommendation is to put this in openstack-mkcloud:
```
if [[ $want_ironic = 1 ]] ; then
  [[ $nics -lt 2 ]] && nics=2
fi
```

With this you can still override it, but in ironic case the number of nics is always 2 mininum. And this will work for all jobs that enable ironic in the future, not only this one.